cluster/examples/kubernetes/ceph/object-openshift.yaml

```diff
@@ -95,3 +95,14 @@ spec:
     # Configure the pod liveness probe for the rgw daemon
     livenessProbe:
       disabled: false
+
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: rook-ceph-rgw-my-store
+  namespace: rook-ceph
+spec:
+  to:
+    kind: "Service"
```

Comment:
nit: you shouldn't need quotes around the `kind` or `name`, but it's fine either way

Reply:
Done.